src/Calculator/Views/Calculator.xaml.cpp

```diff
@@ -714,3 +714,7 @@ void Calculator::Calculator_SizeChanged(Object ^ /*sender*/, SizeChangedEventArg
     }
 }
 
+::Visibility Calculator::ShouldDisplayHistoryButton(bool isAlwaysOnTop, bool isProgrammer, ::Visibility dockPanelVisibility)
+{
+    return !isAlwaysOnTop && !isProgrammer && dockPanelVisibility == ::Visibility::Collapsed ? ::Visibility::Visible : ::Visibility::Collapsed;
```

Comment:
Is this the only mode where we don't want this button to be shown? I wonder if we should consider the scenario where this button appears in other modes it shouldn't either.

Reply:
Yes. The history button should only appear when:
- in Normal mode and scientific mode, when the panel is not displayed and when not in always in top